.github/workflows/cla-pr.yml

```diff
@@ -1,4 +1,4 @@
-name: "CLA Assistant"
+name: "Validate CLA Signature - PR"
```

Comment:
Why do we need this workflow and also `cla-assistant.yml`?

Wouldn't it be simpler to:
1. Eliminate this workflow
2. Update `cla-assistant.yml` to be triggered when issue comments are synchorized
3. Keep `cla-assistant.yml` allowing being invoked externally (`workflow_dispatch`)?
?

Then `cla-assiant.yml` would end up something like the example below:

```yaml
name: "CLA Assistant"
on:
  issue_comment:
    - synchronize
  workflow_call:
```

Reply:
@jfoliveira agreed, could you review the changes again?